pkg/stis/lib/cs1/do2d.c

```diff
@@ -349,28 +347,33 @@ int ngood_extver   io: incremented unless the current imset has zero
 
 	/* Fill in the error array, if it currently contains all zeros. */
 	if (sts->noisecorr == PERFORM) {
-	    if ((status = doNoise (sts, x, &done)))
-		return (status);
-	    if (done) {
-		if (*ngood_extver == 1) {
-		    if ((status = noiseHistory (x->globalhdr)))
+		if ((status = doNoise (sts, x, &done)))
```

Comment:
Can we go back to 4 space tabs here?

Reply:
Maybe it's me, the diff page had 8 space indents here...

Reply:
No, I think it is "2 tabs" that GH is showing as 8 spaces. If you highlight it with mouse, it will be obvious.

Reply:
I am not opposed to changing all tabs to spaces but it will significantly increase the diff.

Reply:
Looks like the code in many places has a mixture of spaces and tabs - 4 spaces for small indents, tabs for large (assumed 8 space) indents.  I can't say I'm a fan of this but as you say fixing it would make the diffs in this PR overwhelming.  Perhaps address it in a separate issue if others don't like the mix of tabs and spaces?  Or is it just me?